diff -urN seyon-2.20c.org/SeDecl.h seyon-2.20c/SeDecl.h
--- seyon-2.20c.org/SeDecl.h	2003-08-03 21:28:41.000000000 +0200
+++ seyon-2.20c/SeDecl.h	2003-08-03 21:29:45.000000000 +0200
@@ -66,7 +66,6 @@
 extern FILE    *open_file(),
                *open_file_va();
 extern void     send_break(),
-                usleep(),
                 show(),
                 showf(),
                 SeyonMessage(),
diff -urN seyon-2.20c.org/SeParse.y seyon-2.20c/SeParse.y
--- seyon-2.20c.org/SeParse.y	2003-08-03 21:28:41.000000000 +0200
+++ seyon-2.20c/SeParse.y	2003-08-03 21:32:18.000000000 +0200
@@ -51,55 +51,4 @@
   (*callbackProc)(4, msg);
 }
 
-#ifdef TEST
-void SignalBeginFunction(char *name)
-{
-  printf("** Function call: %s(", name);
-}
-
-void SignalArg(char *arg)
-{
-  char *p = arg;
-  printf("\n++Arg: (");
-  while (*p) {
-    if (isprint(*p))
-      putchar(*p);
-    else 
-      printf("\\0%o", (int)*p);
-    p++;
-  }
-  putchar(')');
-}
-
-void SignalEndFunction()
-{
-  printf("\n)\n");
-}
-
-void
-main(int argc, char *argv[])
-{
-  char long_line[1000];
 
-  char input_str[] = "This(is, a, real, funky); script();
-            Scripts(); Can(be); Multi(Line, \"Can't they?\");
-            Commas(are, no, longer, optional, inside, arglists);
-	    Scripts(); Can(); contain(\"tabs \\t and backspaces \\b\");
-	    As(\"Well\\ as Quoted Strings\", and, '\"Quoted Strings inside
-	    quoted strings\"');
-	esc(can, appear, outside, strings, ^z, \\012\\015\\n);
-	    But(parenthesis, should, match);
-  We(\"have a funny way of specifying \\012 chars and even)\"); 
-	backslashes( \" \\\\ \");
-  new(\"in this version are ^m and ^A ctr-escapes, as in ^S^Q\");
- The(next, line, will, give, a, syntax, error, because, it, has, two, adj, functions,
-	without, a, separating, semicolon);
- End() script()";
-
-  printf("------ String to parse: \n%s\n\n---- Parsing begins:\n", input_str);
-  strcpy(long_line, input_str);
-  ParseThis(long_line);
-  strcpy(long_line, input_str);
-  ParseThis(long_line);
-}
-#endif TEST
diff -urN seyon-2.20c.org/SeScan.c seyon-2.20c/SeScan.c
--- seyon-2.20c.org/SeScan.c	2003-08-03 21:28:41.000000000 +0200
+++ seyon-2.20c/SeScan.c	2003-08-03 21:30:13.000000000 +0200
@@ -287,13 +287,3 @@
   return GetNextWord();
 }
 
-#ifdef TEST
-main()
-{
-  scSetInputBuf("Just to see if we'\\'re \\n\\033 able to distinguish' words and strings
-\"Also 'quotes' inside strings\" and 'strs \"inside quotes\"'
-Not to forget ^S and ^q control ^ chars");
-
-  while (lGetWord() != 0);
-}
-#endif
